File: RepoTransBench/source_projects/Java/madvirus_ddd-start/src/test/java/net/jodah/typetools/TypeDescriptorTest.java
package net.jodah.typetools;

import org.junit.jupiter.api.Test;
import java.lang.reflect.Method;

import static org.junit.jupiter.api.Assertions.*;

class TypeDescriptorTest {
    @Test
    void testPrimitiveTypesAreSingletons() {
        assertNotNull(TypeDescriptor.BOOLEAN_TYPE);
        assertNotNull(TypeDescriptor.BYTE_TYPE);
        assertNotNull(TypeDescriptor.CHAR_TYPE);
        assertNotNull(TypeDescriptor.DOUBLE_TYPE);
        assertNotNull(TypeDescriptor.FLOAT_TYPE);
        assertNotNull(TypeDescriptor.INT_TYPE);
    }
}

File: RepoTransBench/target_projects/Java/Go/madvirus_ddd-start/tests/original/type_descriptortest.go
package original

import (
	"testing"

	"github.com/stretchr/testify/assert"
)

// TypeDescriptor emulation for test illustration
type TypeDescriptor struct{}

var (
	BOOLEAN_TYPE *TypeDescriptor = &TypeDescriptor{}
	BYTE_TYPE    *TypeDescriptor = &TypeDescriptor{}
	CHAR_TYPE    *TypeDescriptor = &TypeDescriptor{}
	DOUBLE_TYPE  *TypeDescriptor = &TypeDescriptor{}
	FLOAT_TYPE   *TypeDescriptor = &TypeDescriptor{}
	INT_TYPE     *TypeDescriptor = &TypeDescriptor{}
)

func TestPrimitiveTypesAreSingletons(t *testing.T) {
	assert.NotNil(t, BOOLEAN_TYPE)
	assert.NotNil(t, BYTE_TYPE)
	assert.NotNil(t, CHAR_TYPE)
	assert.NotNil(t, DOUBLE_TYPE)
	assert.NotNil(t, FLOAT_TYPE)
	assert.NotNil(t, INT_TYPE)
}